docs/ProgrammingCryptol/highAssurance/HighAssurance.tex

```diff
@@ -174,6 +174,51 @@ \subsection{Capturing test vectors}
   sure the tweak to the above example (removal of dummy single formal
   parameter) works.}
 
+%~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+\subsection{Generating test vectors}
+\label{sec:gentestvec}
+
+Cryptol has the ability to generate test vectors for simple function types using the \texttt{:dumptests} REPL command.
+When \texttt{:dumptests} is given a file to dump the output and a function in scope, it will randomly generate inputs for the function and run them for you.
+The output will contain a tab delimited file in which the first column contains the output, and each column after contains the inputs to the function.
```

Comment:
suggestion: I assume the inputs are in the same order as they are passed to the function?